File: aspnet-mvc-demo/AspnetMvcDemo/Controllers/Pages/ArticlesController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace AspnetMvcDemo.Controllers.Pages
{
    public class ArticlesController : Controller
    {

         public ActionResult Edit()
        {
            return View("~/Views/Pages/Articles/Edit.cshtml");
        }

         public ActionResult List()
        {
            return View("~/Views/Pages/Articles/List.cshtml");
        }

    }
}

File: laravel-demo/app/Http/Controllers/Pages/ArticlesController.php
<?php

namespace App\Http\Controllers\Pages;

use App\User;
use App\Http\Controllers\Controller;

class ArticlesController extends Controller
{
    public function edit()
    {
        return view('pages.articles.edit', ['title' => 'Edit article - Pages']);
    }

    public function list()
    {
        return view('pages.articles.list', ['title' => 'Article list - Pages']);
    }

}
